README.md

```diff
@@ -69,5 +69,8 @@ These extra files and customisation can be found in the
 1. Update the [swagger file](./swagger-v1alpha1.yml) with latest specification.
 2. Update [config file](./config/python.json) to bump the package version.
 3. Run `make python` to re-generate the sdk api files from new specifications.
+
+    **Note** : For Seldon Deploy model metadata service, we are having to patch two additional dict type parameters namely `tags` and `metrics` to the generated docs and sdk code. We do this manually, using the same make command, by applying the [patch file](./templates/python/metadata_tags_metrics.patch) file to the code, post sdk generation task. In some cases the patch apply cmd may fail, and there might be a requirement to re-generate the patch manually by changing the generated docs/sdk files as required.
```

Comment:
@SachinVarghese Can we move this to the "Templates" section of the readme. This we give more detail in future if we need,
And here after step 3 - the note can be if you get "error: patch failed" then look at updating the custom logic.

Reply:
Updated now [6031c75](https://github.com/SeldonIO/seldon-deploy-sdk/pull/48/commits/6031c75486b07088d30b83201430b0556c4b6a15)